test/gpu/native/complexFunctions.chpl

```diff
@@ -0,0 +1,46 @@
+// ensure that complex functions and math can be done using the GPU locale model
+// using complex in a kernel is tested by `test/gpu/native/mathOps.chpl`
```

Comment:
Shouldn't you have uncommented the pertinent lines in the `mathOps` test?

Reply:
No, because `mathOps` is about `complex` in a kernel. Which still doesn't work even with this PR

Reply:
Oh, OK. I remembered you mentioning that maybe GPU support for complex would just fall out from this fix, so seeing this comment made me think that that was the case.

It is OK if it isn't. But could you clarify the comment? If you want, you can just mention that this test is to make sure that the complex functions work on CPU without referring to this test.

It'd be also good to move this test under `test/gpu/nongpu`.